perun-openapi/openapi.yml

```diff
@@ -520,6 +520,16 @@ components:
         memberVos: { type: array, items: { $ref: '#/components/schemas/Vo' } }
         parentVos: { type: array, items: { $ref: '#/components/schemas/Vo' } }
 
+    ConsentHub:
+      allOf:
+        - $ref: '#/components/schemas/Auditable'
+        - properties:
+            name: { type: string }
+            enforceConsents: { type: boolean }
+            facilities: { type: array, items: { $ref: '#/components/schemas/Facility' } }
+      discriminator:
+        propertyName: beanName
+
```

Comment:
Maybe there should be some of these properties required? I think that `name` and `facilities` should be required, `enforceConsents` has a default value, so it can be optional.

Reply:
Good point; I made them all required.